statistics/handle/handle.go

```diff
@@ -741,36 +760,52 @@ func (h *Handle) mergePartitionStats2GlobalStats(sc sessionctx.Context,
 		}
 		// If pre-load partition stats isn't provided, then we load partition stats directly and set it into allPartitionStats
 		if allPartitionStats == nil || partitionStats == nil || !ok {
-			partitionStats, err = h.loadTablePartitionStats(tableInfo, &def)
-			if err != nil {
+			var err1 error
+			partitionStats, err1 = h.loadTablePartitionStats(tableInfo, &def)
+			if err1 != nil {
+				if skipMissingPartitionStats && types.ErrPartitionStatsMissing.Equal(err) {
+					globalStats.MissingPartitionStats = append(globalStats.MissingPartitionStats, fmt.Sprintf("partition `%s`", def.Name.L))
+					continue
+				}
+				err = err1
 				return
 			}
 			if allPartitionStats == nil {
 				allPartitionStats = make(map[int64]*statistics.Table)
 			}
 			allPartitionStats[partitionID] = partitionStats
 		}
+		globalStats.Count += partitionStats.Count
+		globalStats.ModifyCount += partitionStats.ModifyCount
```

Comment:
Do we need to remove the count adding later in this func?

Reply:
Yes. It is a mistake I made when resolving conflicts.